src/backup_restore.cpp

```diff
@@ -23,4 +162,9 @@ void BackupRestore::backupAndRestoreIpzVpd(types::IPZVpdMap& io_srcVpdMap,
     (void)i_srcPath;
     (void)i_dstPath;
 }
+
+void BackupRestore::setIsBackupAndRestoreCalled(bool i_isCalled)
```

Comment:
isCalled does not seem like a proper name.

Reply:
Updated